package.json

```diff
@@ -63,7 +64,11 @@
     "test:single": "mocha -timeout 180000 --full-trace",
     "test:system": "mocha -timeout 180000 --recursive --full-trace system_test/*.js",
     "test:unit": "mocha -timeout 180000 --recursive --full-trace test/unit/**/*.js test/unit/*.js",
-    "test:ci": "mocha -timeout 180000 --recursive --full-trace test/**/*.js system_test/*.js"
+    "test:ci": "mocha -timeout 180000 --recursive --full-trace test/**/*.js",
+    "test:ci:withSystemTests": "mocha -timeout 180000 --recursive --full-trace test/**/*.js system_test/*.js",
+    "test:unit:coverage": "nyc npm run test",
```

Comment:
let's use `run test:unit` - it's more explicit

Reply:
ok